releases/v21_0_0.rst

```diff
@@ -89,3 +220,80 @@ Removal of the obs_ctio0m9 package
 
 The obs_ctio0m9 camera package will be removed from the Science Pipelines before the release of version 22.0.0.
 :jirab:`RFC-729, DM-26867, DM-26868`
+
+.. _release-v21-0-0-deprecate-gen2tasks_MetricTask:
+
+Removal of MetricTask
+---------------------
+
+The :code:`MetricTask` is replaced by `lsst.verify.tasks.MetricTask`.
+It will be removed along with daf_persistence before the next release 22.0.0.
```

Comment:
Say that the `MetricTask` is from `lsst.verify.gen2tasks`?

I don't think daf_persistence will be removed before 22.0.0, as the Gen2 Butler will still need to be present.

"It will be removed before the release of version 22.0.0."

Reply:
Correct, but the original comment says: "To be removed along with daf_persistence." 

So I think it would be good to keep the same in the release note: "It will be removed along with daf_persistence".

Reply:
Then it should move to the "Pending deprecation" section.